packages/fxa-settings/src/components/LocaleToggle/index.tsx

```diff
@@ -31,20 +39,41 @@ export const LocaleToggle: React.FC<LocaleToggleProps> = ({
     'Select language'
   );
 
+  const browserDefaultLabel = ftlMsgResolver.getMsg(
+    'locale-toggle-browser-default',
+    'Browser default'
+  );
+
+  // Get browser's default locale info for functionality (not display)
+  const browserDefaultLocale = getBrowserDefaultLocaleInfo();
+
+  // Determine the current value for the select
+  // If using browser default, show the actual browser locale in the dropdown
+  const currentValue = isUsingBrowserDefault 
+    ? (browserDefaultLocale?.code || currentLocale)
+    : currentLocale;
+
   return (
     <div className={`relative ${className}`}>
```

Comment:
I don't think you need the `relative` class on this.

Also just a suggestion, but if you add these classes:

```
bg-grey-10 p-1 tablet:bg-transparent tablet:p-0 rounded-md border border-grey-50 tablet:border-none
```

Then mobile looks a bit better:
<img width="400" alt="image" src="https://github.com/user-attachments/assets/164df936-f71a-48c7-9098-89937f5e6bd2" />

However, this makes me wonder if we want this to be in a `fixed` position at all. Should it just follow our same `<footer>` standards as what we have inside Settings, where it stays at the bottom?

<img width="600" height="378" alt="image" src="https://github.com/user-attachments/assets/bfc28c71-5d72-416b-a456-774fdc342ae5" />